source/data-formats/decimal128.txt

```diff
@@ -15,11 +15,10 @@ Decimal128
 Overview
 --------
 
-MongoDB 3.4 introduced support for a Decimal128 BSON type,
-which is a 128-bit decimal-based
-floating-point value capable of emulating decimal rounding with exact precision.
-This functionality is intended for applications that handle :manual:`monetary
-data </tutorial/model-monetary-data>`, such as financial and tax computations.
+Decimal128 BSON type is a 128-bit decimal-based floating-point value capable 
```

Comment:
Should this say "The Decimal128 BSON type"? This reads as if "Decimal128 BSON type" is a proper noun.